packages/router/src/resources/viewport.ts

```diff
@@ -1,10 +1,10 @@
-import { Constructable, InterfaceSymbol } from '@aurelia/kernel';
-import { bindable, CustomElementResource, INode, LifecycleFlags } from '@aurelia/runtime';
+import { Constructable, IContainer, InterfaceSymbol, Writable } from '@aurelia/kernel';
+import { bindable, createRenderContext, CustomElementResource, IDOM, IElementTemplateProvider, INode, IRenderContext, IRenderingEngine, ITemplate, LifecycleFlags, TemplateDefinition } from '@aurelia/runtime';
 import { Router } from '../router';
 import { IViewportOptions, Viewport } from '../viewport';
 
 export class ViewportCustomElement {
-  public static readonly inject: ReadonlyArray<InterfaceSymbol|Constructable> = [Router, INode];
+  public static readonly inject: ReadonlyArray<InterfaceSymbol<unknown> | Constructable> = [Router, INode, IRenderingEngine];
```

Comment:
```suggestion
  public static readonly inject: ReadonlyArray<InterfaceSymbol|Constructable> = [Router, INode, IRenderingEngine];
```
Not sure why this was changed back, but this means the same and is terser